scripts/ivy/generate-blacklist.js

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
+
+const path = require('path');
+const fs = require('fs');
+
+const karmaOutput = JSON.parse(fs.readFileSync('/tmp/karma-result.json'));
```

Comment:
Could probably make this as a CLI arg as well. I'm sure it doesn't matter too much, so just nit.